### ISSUE TITLE
Website doesn't have keyboard focus when closing/undoing single tab

### ISSUE THREAD

**Issue by The-Compiler** [qutebrowser/qutebrowser#6532]
**Version info**:
qutebrowser v2.2.3
Git commit: 96149c7e2 on master (2021-06-09 10:22:17 +0200)
Backend: QtWebEngine 5.15.4, based on Chromium 87.0.4280.144
Qt: 5.15.2

**Does the bug happen if you start with `--temp-basedir`?**: yes

**How to reproduce**
`qutebrowser --temp-basedir -s tabs.last_close blank -s tabs.show multiple ':later 2000 tab-close' ':later 4000 undo'`

and pressing tab won't switch focus between elements on the page.
